moveit_ros/visualization/rviz_plugin_render_tools/src/planning_link_updater.cpp

```diff
@@ -35,7 +35,7 @@
 /* Author: Ioan Sucan */
 
 #include <moveit/rviz_plugin_render_tools/planning_link_updater.h>
-#include <OgreVector3.h>
+#include <Ogre.h>
 #include <OgreQuaternion.h>
```

Comment:
Can you check if Ogre.h includes OgreQuaternion.h?

Reply:
It does not include it directly, but it does include OgreVector.h which includes OgreQuaternion.h

Reply:
Let's remove OgreQuaternion.h from here, then